backend/ches/ChesService.cs

```diff
@@ -224,8 +224,8 @@ public async Task<EmailResponseModel> SendEmailAsync(IEmail email)
 
             // Make sure there are no blank CC or BCC;
             email.To = email.To.NotNullOrWhiteSpace();
-            email.Cc = email.Cc.NotNullOrWhiteSpace();
-            email.Bcc = email.Bcc.NotNullOrWhiteSpace();
+            email.Cc = email.Cc?.NotNullOrWhiteSpace();
```

Comment:
Fix a bug when passing null values to these properties.

Reply:
bcc and cc get this but not To? is to always provided?

Reply:
If they don't provide a `to` it would have thrown an argument exception earlier.